fmriprep/cli/run.py

```diff
@@ -124,16 +124,20 @@ def get_parser():
                              '6 degrees (rotation and translation) are used by default.')
     g_conf.add_argument(
         '--output-space', required=False, action='store',
-        choices=['T1w', 'template', 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
-        nargs='+', default=['template', 'fsaverage5'],
-        help='volume and surface spaces to resample functional series into\n'
-             ' - T1w: subject anatomical volume\n'
-             ' - template: normalization target specified by --template\n'
-             ' - fsnative: individual subject surface\n'
-             ' - fsaverage*: FreeSurfer average meshes\n'
-             'this argument can be single value or a space delimited list,\n'
-             'for example: --output-space T1w fsnative'
-    )
+        choices=['T1w', 'MNI', 'template', 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
+        nargs='+', default=['MNI', 'fsaverage5'],
+        help="""\
+volume and surface spaces to resample functional series into
+ - T1w: subject anatomical volume
+ - MNI: spatial normalization using the target MNI template specified by --template
```

Comment:
Could you talk about this decision to change `template` to `MNI`? Are we refusing to support templates in non-MNI space (except FreeSurfer)? How will this affect attempts to support developmental and/or non-human populations? If this is a move to be more in line with derivatives discussion of templates/spaces should we be changing the FreeSurfer options to `--output-space FS` and use `--template fsaverage{,5,6}`?

Reply:
Yes, I'm inclined to think that ``--output-space`` and ``--template`` will be merged in one argument (something like ``--output-reference``) that will take values like ``orig``, ``T1w``, ``MNI152{Lin,NlinAsym2009c}``, ``fsaverage{,5,6}`` and probably ``custom`` so then the template may be passed by path.

In the contrary to my views in the realm of BEP014 as regards space/coordinates/templates definitions, I believe that here we must be explicit about the reference (i.e. prefer just ``MNI<mni-id>`` over the combination of ``--output-space MNI --template MNI<mni-id>``). However this seemed like a larger change that will also require arranging more templates in templateflow (and also the archive to keep it in sync since we are not yet accessing the templates through datalad).

Reply:
@effigies should we go for it in this PR? Do you even like the idea? WDYT?

Reply:
BTW - https://neurostars.org/t/how-to-keep-fmriprep-output-space-to-native-epi/2679/3?u=oesteban